src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -681,6 +686,7 @@ private function handlePlayerActionFromData(int $action, BlockPosition $blockPos
 
 		switch($action){
 			case PlayerAction::START_BREAK:
+			case PlayerAction::CONTINUE_DESTROY_BLOCK: //destroy the next block while holding down left click
```

Comment:
CONTINUE_DESTROY_BLOCK is also called when changing to another face while breaking the same block which would result in a new SurvivalBlockBreakHandler being constructed for something that's already being broken

Reply:
Ok, good call I wasn't aware. Will verify

Reply:
Verified the behaviour of this matches vanilla (bug for bug too).